components/user/EventRow.tsx

```diff
@@ -29,6 +29,9 @@ import {
 } from 'utils/events'
 
 import styles from './EventRow.module.css'
+import ActivityMaspMint from 'components/icons/ActivityMaspMint'
+import ActivityMaspBurn from 'components/icons/ActivityMaspBurn'
+import ActivityMaspTransfer from 'components/icons/ActivityMaspTransfer'
```

Comment:
can we put these next to the other icons up top?

Reply:
Yup, this auto imported this way, but I can move em